tests/testapps/CTCatalog/CTCatalog/Tests/CTCFrameTestViewController.mm

```diff
@@ -27,6 +26,11 @@ @interface CTFrameLineTestView : UIView {
 @implementation CTFrameLineTestView {
 }
 
+- (void)setLineRef:(CTLineRef)lineRef {
```

Comment:
No-can-do: CT* are not NSObject descendents; pending #719, when they will be.

Retain properties require `__attribute__((NSObject))`, which in turn requires `objc_retain` and `objc_release` conformance.

---
In reply to: [79957106](https://github.com/Microsoft/WinObjC/pull/1010#discussion_r79957106) [](ancestors = 79957106)